tests/basic/storage_pool_test.go

```diff
@@ -5368,6 +5390,42 @@ var _ = Describe("{PoolResizeVolumesResync}", func() {
 	})
 })
 
+// This function checks for pool status on selectedNode and when the pool goes offline it will expand the pool with ID poolID with the given type of expand and to expected size
+func poolStatusChecker(done *chan bool, errorChan *chan error, selectedNode node.Node, PoolID string, expectedSize uint64, isjournal bool, poolExpandLock *sync.Mutex) {
```

Comment:
// poolStatusChecker checks for pool status....

Reply:
might panic if done, errorChan, and poolExpandLock are nil

Reply:
Changed